### PR TITLE
(FM-4046) Update to current msync configs [006831f]

### PR DESCRIPTION
**Pull request by DavidS**

### CODE REVIEW

**Comment by daenney** [comment]
Oi oi oi. You can't just override the license like that :scream:.

**Comment by DavidS** [comment]
The original "License" statement moved to NOTICE, in accordance to the ASF's guidelines for applying Apache-2.0. This doesn't change the legal status of anything as far as I can tell.

I should probably add some words to that effect in the commit message.

**Comment by daenney** [comment]
Oh, you're absolutely right. I missed that.

**Comment by DavidS** [comment]
This commit message better?

**Comment by daenney** [comment]
Yes, thank you! :heart:

**Comment by alexjfisher** [comment]
@DavidS But what happened to the MIT license bit??

I'm trying to figure out what should be in the metadata.json in puppet-unattended_upgrades.  See
https://github.com/voxpupuli/puppet-unattended_upgrades/issues/93

File: LICENSE [hunk 1]
```diff
@@ -1,34 +1,202 @@
-Copyright (c) 2011 Evolving Web Inc.
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
```
Comment (alexjfisher) [inline LICENSE]: This is the MIT license though, isn't it?

Comment (DavidS) [inline LICENSE]: Apache-2.0, which the latter contributions through Puppet Labs, Inc., were licensed is more restrictive than the original MIT, it makes sense to have it in the main file. Attributions are still preserved in the NOTICE file, which is recommended by the Apache 2.0 application guidelines.

**Pull request merged**